### ISSUE TITLE
Spec and PDF link broken

### ISSUE THREAD

**Issue by danivideda** [IntersectMBO/cardano-ledger#3196]
There seems to be a problem with the link in the README file. I can't access the [PDF](https://hydra.iohk.io/job/Cardano/cardano-ledger/blocksCDDLSpec/latest/download-by-type/doc-pdf/binary) and [Spec](https://hydra.iohk.io/job/Cardano/cardano-ledger/shelleyLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec) link like I used to. It just gives me a blank page, and with different browser (Brave), it shows Error 418
![image](https://user-images.githubusercontent.com/57628386/205842107-7565bb37-02d6-4b8a-a4f2-fefd8052c2cc.png)

**Comment by lehins** [comment]
Our CI has very recently migrated to a new service. We need to update all the links in the repo.
Thank you for reporting this.

**Comment by JaredCorduan** [comment]
See https://github.com/input-output-hk/cicero/issues/67

**Comment by eyeinsky** [comment]
For those ending up here, it's possible to build the pdfs locally by cloning the repo, then:
```
# install texlive with your package manager

# make pdfs
find . -path '*/formal-spec' -type d | while read dir; do (cd $dir && make all); done

# list results
find . -path '*/formal-spec/*.pdf' -type f | grep -v rules.pdf # remove rules.pdf as that is a commited file
```

**Comment by JaredCorduan** [comment]
Resolved by the ever-awesome @teodanciu in #3245 and #3260 :tada:
